docs/setup/upgrade/resolving-migration-failures.asciidoc

```diff
@@ -123,3 +123,64 @@ In {kib} 7.5.0 and earlier, when the task manager index is set to `.tasks`
 with the configuration setting `xpack.tasks.index: ".tasks"`,
 upgrade migrations fail. In {kib} 7.5.1 and later, the incompatible configuration
 setting prevents upgrade migrations from starting.
+
+[float]
+==== Repeated time-out requests that eventually fail
+
+Migrations get stuck in a loop of retry attempts waiting for index yellow status that's never reached. If you see a log entry similar to `Action failed with "Request timed out". Retrying attempt 1 in 2 seconds.` in either the CLONE_TEMP_TO_TARGET or CREATE_REINDEX_TEMP steps, it's because the process is waiting for an index yellow status.
```

Comment:
It might be a little easier to read if you restructure this sentence and pull out the error message into a code box like this:

In the `CLONE_TEMP_TO_TARGET or CREATE_REINDEX_TEMP` steps, you might see a log entry similar to:

> Action failed with "Request timed out". Retrying attempt 1 in 2 seconds.

The process is waiting for a yellow index status.  There are two known causes: